rotkehlchen/globaldb/cache.py

```diff
@@ -70,6 +70,7 @@ def globaldb_get_general_cache_keys_and_values_like(
     """
     Function to read globaldb cache.
     Returns all pairs key-value where key starts with the provided `key_parts`.
+
     `key_parts` should not contain neither `%` nor `.` symbols.
```

Comment:
That's not what I had asked. Look at the review carefully.

https://github.com/rotki/rotki/pull/5807#discussion_r1152145274

` key_parts should contain neither the "%" nor the "." symbol.`

What you have writen there is double negation. Both here and in the other function please.